ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineForkchoiceUpdatedV3.java

```diff
@@ -48,14 +49,21 @@ public String getName() {
 
   @Override
   protected ValidationResult<RpcErrorType> validateParameter(
-      final EngineForkchoiceUpdatedParameter fcuParameter) {
+      final EngineForkchoiceUpdatedParameter fcuParameter,
+      final Optional<EnginePayloadAttributesParameter> maybePayloadAttributes) {
     if (fcuParameter.getHeadBlockHash() == null) {
       return ValidationResult.invalid(RpcErrorType.INVALID_PARAMS, "Missing head block hash");
     } else if (fcuParameter.getSafeBlockHash() == null) {
       return ValidationResult.invalid(RpcErrorType.INVALID_PARAMS, "Missing safe block hash");
     } else if (fcuParameter.getFinalizedBlockHash() == null) {
       return ValidationResult.invalid(RpcErrorType.INVALID_PARAMS, "Missing finalized block hash");
     }
+    if (maybePayloadAttributes.isPresent()) {
```

Comment:
this is the parameter checking you mean that will potentially conflict with the larger refactor?

Reply:
I think fine to fix it like this in the short-term to clear the noise for the hive tests

Reply:
We would not necessarily have to validate this if the plans are to use strict JSON validation